modules/tinymce/src/core/main/ts/api/SettingsTypes.ts

```diff
@@ -12,7 +12,8 @@ import { AllowedFormat } from './fmt/StyleFormat';
 import { SchemaType } from './html/Schema';
 import { EditorUiApi } from './ui/Ui';
 
-export type EntityEncoding = 'named' | 'numeric' | 'raw';
+// Can take a list of named / numeric / raw values, delimited by '+' or ','
+export type EntityEncoding = 'named' | 'numeric' | 'raw' | string;
```

Comment:
Adding `string` here just dwarfs the others and removes all type safety. We need to come up with a better solution here.